_glossary/index.md

```diff
@@ -308,7 +308,7 @@ A new declaration of a local variable _shadows_ one of the same name in an enclo
 _Signature_ is short for [type signature](#type-signature).
 
 * #### singleton object
-An object defined with the object keyword. Each singleton object has one and only one instance. A singleton object that shares its name with a class, and is defined in the same source file as that class, is that class’s [companion object](#companion-object). The class is its [companion class](#companion-class). A singleton object that doesn’t have a companion class is a [standalone object](#standalone-object).
+An object defined with the object keyword. Each singleton object has one and only one instance. A singleton object that shares its name with a class, and is defined in the same source file as that class, is that class’s [companion object](#companion-object). The class is its [companion class](#companion-class). A singleton object that doesn't have a companion class is a [standalone object](#standalone-object).
```

Comment:
why change this?

Reply:
I tried to somehow unify it because there are two different symbols present in documentation, like `'` and `’`.
 I'll remove that type of changes